### PR TITLE
TE-626 Standardise amenities array in component

### PR DESCRIPTION
**Pull request by stephenkr**
[Related YouTrack issue](https://youtrack.lodgify.net/issue/TE-626)

### What **one** thing does this PR do?
Standardise amenities array in RoomType component

### Any other notes
And add a prop to amenities to determine if the amenity items should be stacked

screen shot 2018-07-18 at 18 10 01
screen shot 2018-07-18 at 18 10 07

### CODE REVIEW

File: src/components/property-page-widgets/Amenities/Readme.md [hunk 1]
```diff
@@ -114,4 +114,29 @@ const amenities = [
 
 <Amenities amenities={amenities} headingText="Property Amenities" />
 
+```
+
+#### Display Amenities as stacked
```
Comment (marcdavi-es) [inline src/components/property-page-widgets/Amenities/Readme.md]: Please make this (and the rest of the file) consistent with other examples.

The structure is probably

```md
[default example]

### Content

#### Heading

[heading example]

### Variations

#### Stacked

[stacked example]

```

File: src/components/property-page-widgets/Amenities/component.js [hunk 2]
```diff
@@ -61,12 +61,6 @@ Component.propTypes = {
   ).isRequired,
   /** The text to display as a heading at the top of the amenities. */
   headingText: PropTypes.string,
-  /**
-   * Is the user on a mobile device.
-   * Provided by `withResponsive` so ignored in the styleguide.
-   * @ignore
-   */
-  isUserOnMobile: PropTypes.bool.isRequired,
+  /** Should the amenities appear as stacked */
```
Comment (marcdavi-es) [inline src/components/property-page-widgets/Amenities/component.js]: 1. Please make consistent with other Boolean prop documentation. I don't think we use `Should` anywhere. `Is` or `Are` please.
2. I'm not sure this helps to understand what 'stacked' means. Please be clearer e.g. `Are the amenities displayed stacked on top of one another`

File: src/components/property-page-widgets/RoomType/component.js [hunk 3]
```diff
@@ -150,10 +150,10 @@ Component.propTypes = {
...
       iconName: PropTypes.string.isRequired,
-      /** Is the amenity disabled. */
-      isDisabled: PropTypes.bool,
+      /** The list of amenities */
+      items: PropTypes.arrayOf(PropTypes.string.isRequired).isRequired,
       /** A visible label to display for the amenity. */
-      labelText: PropTypes.string.isRequired,
+      name: PropTypes.string.isRequired,
     })
   ).isRequired,
```
Comment (marcdavi-es) [inline src/components/property-page-widgets/RoomType/component.js]: Please check that the documentation for the `amenities` prop as a whole is consistent with the documentation for `Amenities`.

Line 145 vs line 153 is confusing as they both now claim to be a list of amenities. Line 145 should document categories of amenities.

File: src/components/property-page-widgets/Amenities/Readme.md (hunk 1 above)
Comment (stephenkr) [reply src/components/property-page-widgets/Amenities/Readme.md]: Will do.

File: src/components/property-page-widgets/Amenities/component.js (hunk 2 above)
Comment (stephenkr) [reply src/components/property-page-widgets/Amenities/component.js]: Yep ok, i'll make it more clear.

File: src/components/property-page-widgets/RoomType/component.js (hunk 3 above)
Comment (stephenkr) [reply src/components/property-page-widgets/RoomType/component.js]: Oh true, i'll improve the comments.

**Pull request merged**